### PR TITLE
Fix docker run command

### PR DESCRIPTION
**Pull request by hardillb**
Swap `-n` for `--name`

### CODE REVIEW

**Review by Steve-Mcl — APPROVED** [review]
yes, that does make it clearer Ben.

**Comment by Steve-Mcl** [comment]
Actually ben, seems there is another issue with the docs.

They say to set the below in `mosquitto.conf `...

```
    auth_opt_http_getuser_uri /api/broker/auth-client
    auth_opt_http_aclcheck_uri /api/broker/auth-acl
```

However after a fair bit of debugging (in the wrong places ;) ) - I have found these should be ...

```
auth_opt_http_getuser_uri /api/comms/auth/client
auth_opt_http_aclcheck_uri /api/comms/auth/acl
```

**Review by Steve-Mcl — CHANGES_REQUESTED** [review]
Ben, could lines 105, 106 & 143, 144 be changed from 

```
    auth_opt_http_getuser_uri /api/broker/auth-client
    auth_opt_http_aclcheck_uri /api/broker/auth-acl
```

to 

```
    auth_opt_http_getuser_uri /api/comms/auth/client
    auth_opt_http_aclcheck_uri /api/comms/auth/acl
```

**Comment by hardillb** [comment]
@Steve-Mcl updated

File: docs/install/local/README.md [hunk 1]
```diff
@@ -102,8 +102,8 @@ Once installed, you will need to build and install the authentication plugin.
     auth_opt_acl_jitter_seconds 5
     auth_opt_http_host localhost
     auth_opt_http_port 3000
-    auth_opt_http_getuser_uri /api/broker/auth-client
-    auth_opt_http_aclcheck_uri /api/broker/auth-acl
+    auth_opt_http_getuser_uri /api/comms/auth-client
+    auth_opt_http_aclcheck_uri /api/comms/auth-acl
```
Comment (knolleary) [inline docs/install/local/README.md:106]:
```suggestion
    auth_opt_http_getuser_uri /api/comms/auth/client
    auth_opt_http_aclcheck_uri /api/comms/auth/acl
```

File: docs/install/local/README.md [hunk 2]
```diff
@@ -140,17 +140,17 @@ you can use a pre-built docker image that provides everything needed.
     auth_opt_acl_jitter_seconds 5
     auth_opt_http_host 172.17.0.1
     auth_opt_http_port 3000
-    auth_opt_http_getuser_uri /api/broker/auth-client
-    auth_opt_http_aclcheck_uri /api/broker/auth-acl
+    auth_opt_http_getuser_uri /api/comms/auth-client
+    auth_opt_http_aclcheck_uri /api/comms/auth-acl
```
Comment (knolleary) [inline docs/install/local/README.md:144]:
```suggestion
    auth_opt_http_getuser_uri /api/comms/auth/client
    auth_opt_http_aclcheck_uri /api/comms/auth/acl
```

**Review by knolleary — CHANGES_REQUESTED** [review]
Fixed auth paths

**Review by Steve-Mcl — APPROVED** [review]
Thanks Ben

**Pull request merged**